unity-renderer/Assets/Scripts/MainScripts/DCL/Environment/TestEnvironment/ServiceLocatorFactory.cs

```diff
@@ -0,0 +1,53 @@
+﻿using DCL.Controllers;
+using DCL.Helpers.NFT.Markets;
+using DCL.Rendering;
+using NSubstitute;
+
+namespace DCL.Tests
+{
+    public static class ServiceLocatorFactory
```

Comment:
I find both ServiceLocatorFactories unneded, if we are going to keep them we should at least rename this one so we don't have a name collision. Although the namespace prevent issues, it might create unnecessary headaches or problems with IDEs

Reply:
Renamed and collapsed the namespace.